_posts/fr/newsletters/2024-10-11-newsletter_fr.md

```diff
@@ -0,0 +1,270 @@
+
+- [Core Lightning #7539][] ajoute une commande RPC `getemergencyrecoverdata` pour récupérer et
+  retourner les données du fichier `emergency.recover`. Cela permettra aux développeurs utilisant
+  l'API d'ajouter une fonctionnalité de sauvegarde de portefeuille à leurs applications.
+
+- [LDK #3179][] introduit de nouveaux [messages onion][topic onion messages] `DNSSECQuery` et `DNSSECProof`,
+  et un `DNSResolverMessageHandler` pour gérer ces messages comme la fonctionnalité
+  principale pour implémenter [BLIP32][]. Cette PR ajoute également un `OMNameResolver` qui vérifie
+  les preuves DNSSEC et les transforme en [offres][topic offers]. Voir le Bulletin [#306][news306
+  blip32].
+
+- [LND #8960][] implémente une fonctionnalité de canal personnalisé en ajoutant une superposition
+  taproot comme un nouveau type de canal expérimental, qui est identique à un [canal taproot
+  simple][topic simple taproot channels] mais engage des métadonnées supplémentaires dans les feuilles
+  [tapscript][topic tapscript] pour les scripts de canal. La machine d'état principale du canal et la
+  base de données sont mises à jour pour traiter et stocker les feuilles tapscript personnalisées. Une
+  option de configuration `TaprootOverlayChans` doit être définie pour activer le support des canaux de
+  superposition taproot. L'initiative des canaux personnalisés améliore le support de LND pour [les
+  actifs taproot][topic client-side validation]. Voir le Bulletin [#322][news322 customchans].
+
+- [Libsecp256k1 #1479][] ajoute un module [MuSig2][topic musig] pour un schéma multisig compatible
+  avec [BIP340][] tel que spécifié dans [BIP327][]. Ce module est presque identique à celui implémenté
+  dans [secp256k1-zkp][zkpmusig2], mais présente quelques changements mineurs, tels que la suppression
+  du support pour les [signatures adaptatives][topic adaptor signatures] pour le rendre non expérimental.
+
+- [Rust Bitcoin #2945][] introduit le support pour [testnet4][topic testnet] en ajoutant
+  l'énumération `TestNetVersion`, en refactorisant le code, et en incluant les paramètres nécessaires
+  et les constantes de la blockchain pour testnet4.
+
+- [BIPs #1674][] revient sur les modifications de la spécification [BIP85][]
+  décrites dans le [Bulletin #323][news323 bip85]. Les changements ont rompu
+  la compatibilité avec les versions déployées du protocole. La discussion sur
+  la PR a soutenu la création d'un nouveau BIP pour les changements majeurs.
+
+{% assign four_days_after_posting = page.date | date: "%s" | plus: 345600 | date: "%Y-%m-%d 14:30" %}
+{% include snippets/recap-ad.md when=four_days_after_posting %}
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="7494,7539,3179,8960,1479,2945,1674,26898,27610,27608" %}
+[BDK 1.0.0-beta.5]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-beta.5
+[news323 bip85]: /fr/newsletters/2024/10/04/#bips-1600
+[sanders guide]: /en/bitcoin-core-28-wallet-integration-guide/
```

Comment:
@Copinmalin do you plan to translate this blog post to fr?

Reply:
It can be cool yes!

Reply:
We could ! OK lets go

Reply:
Can we include it in this PR or do you plan a separate one?